tests/css-borders-4.js

```diff
@@ -149,49 +186,438 @@ export default {
 			},
 			tests: [
 				'round',
-				'angle',
-				'round angle',
-				'round angle round',
-				'round angle round angle',
+				'scoop',
+				'bevel',
+				'notch',
+				'square',
+				'squircle',
+				'superellipse(4)',
+				'superellipse(infinity)',
+				'round scoop',
+				'round scoop bevel',
+				'round scoop bevel notch',
+			],
+		},
+		'corner-top-left-shape': {
+			links: {
+				dev: '#corner-shaping',
+			},
+			tests: [
+				'round',
+				'scoop',
+				'bevel',
+				'notch',
+				'square',
+				'squircle',
+				'superellipse(4)',
+				'superellipse(infinity)',
+			],
+		},
+		'corner-top-right-shape': {
+			links: {
+				dev: '#corner-shaping',
+			},
+			tests: [
+				'round',
+				'scoop',
+				'bevel',
+				'notch',
+				'square',
+				'squircle',
+				'superellipse(4)',
+				'superellipse(infinity)',
```

Comment:
For these duplicate values, can you define them into a JS variable to manage them consistently?

Reply:
Done. Thanks for the hint!